docs/src/languages.md

```diff
@@ -51,3 +51,20 @@ go get github.com/mxschmitt/playwright-go
 
 * [GitHub repo](https://github.com/mxschmitt/playwright-go)
 * [Documentation](https://pkg.go.dev/github.com/mxschmitt/playwright-go?tab=doc)
+
+## Crystal
```

Comment:
Could you please add this to the showcase instead? I've just moved Go language port there (see #4934). Unfortunately, we cannot put unofficial language ports in this list.

Also, we are currently changing our docs, so there is some churn and everyone has to rebase 😄  Sorry about that!

Reply:
Sure. Do you think it make more sense for me to discard this PR and resubmit a new one?

Reply:
Should be possible in this PR instead of creating a new one. either rebase or revert last commit, stash it, pull from master and pop the stash.

Reply:
> revert last commit, stash it, pull from master and pop the stash.

Or simply do a [`$ git commit --amend`](https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things).

Reply:
Thanks @mxschmitt . I must confess that I might have messed up 😸

Reply:
Apologies, but have raised new PR https://github.com/microsoft/playwright/pull/4945